aya/src/obj/mod.rs

```diff
@@ -1305,6 +1305,45 @@ fn parse_btf_map_def(btf: &Btf, info: &DataSecEntry) -> Result<(String, BtfMapDe
     Ok((map_name.to_string(), map_def))
 }
 
+pub fn parse_map_info(info: bpf_map_info, pinned: bool) -> Map {
```

Comment:
I think this should be moved to `maps/mod.rs`? `obj/` is supposed to be for ELF object related code.

Reply:
Settled above